Client/Frontend/Intro/IntroViewModel.swift

```diff
@@ -46,7 +49,16 @@ struct IntroViewModel: OnboardingViewModelProtocol, FeatureFlaggable {
     }
 
     var enabledCards: [IntroViewModel.InformationCards] {
-        return [.welcome, .signSync]
+        let notificationCardPosition = OnboardingNotificationCardHelper().cardPosition
```

Comment:
Can we add unit test for this logic?

Reply:
This is currently not testable. I created [a task](https://mozilla-hub.atlassian.net/browse/FXIOS-5893) to make Nimbus feature flags like this one testable.